ls1intum/Apollon
# `Feature`: Inverted editor layout

As per Stephan's request, this PR flips the layout of the Apollon Editor to have the component sidebar on the left side of the editor canvas.

### Checklist

### Motivation and Context
To make Apollon more similar to other editor applications, the component sidebar is moved to the left side.

### Steps for Testing
Open up Apollon, see if the sidebar is correctly rendered on the left side of the editor canvas and make sure paddings within the application seem correctly sized.

### Screenshots

<img width="1561" alt="Screenshot 2023-11-19 at 22 25 53" src="https://github.com/ls1intum/Apollon/assets/143808484/07851f93-1912-45e7-9e41-40bc6b7a4af8">
